Automattic/jetpack
# SSO improvements main PR

This is the main PR towards SSO improvements. It's a place to merge PRs into, please do not push anything to it that is not updating it against trunk.


## Proposed changes:
Includes the following PRs:
- https://github.com/Automattic/jetpack/pull/35158
- https://github.com/Automattic/jetpack/pull/35282 
- https://github.com/Automattic/jetpack/pull/35298
- https://github.com/Automattic/jetpack/pull/35395
- https://github.com/Automattic/jetpack/pull/35234
- https://github.com/Automattic/jetpack/pull/35304
- https://github.com/Automattic/jetpack/pull/35473
- https://github.com/Automattic/jetpack/pull/35475
- https://github.com/Automattic/jetpack/pull/35531
- https://github.com/Automattic/jetpack/pull/35572
- https://github.com/Automattic/jetpack/pull/35573
- https://github.com/Automattic/jetpack/pull/35617
- https://github.com/Automattic/jetpack/pull/35683
- https://github.com/Automattic/jetpack/pull/35682
- https://github.com/Automattic/jetpack/pull/35685
- https://github.com/Automattic/jetpack/pull/35675

**Summary**:

 Site owners now have a much better experience managing users.
- Different colors for users connected, not invited or with a pending invite to join WordPress.com
- Possibility to quick send an invite to a user
- Possibility to revoke an invitation
- When adding a new user we now have a custom message sent, a notice that shows to the site owner that an invitation will be sent. And when the user is created, we send a WordPress.com invitation.

### Other information:


## Jetpack product discussion

* Design/Kick-off thread: pdDR7T-1jU-p2
* Project proposal / feedback requested thread: pdDR7T-1ja-p2

## Does this pull request change what data or activity we track or use?



## Testing instructions:






- Sync the Jetpack plugin to your site.
- Check that the summary changes are reflected.
- Try to break this.
- Check how it is on mobile.
